cypress/e2e/pages/onDemandTV/tests.js

```diff
@@ -19,7 +16,7 @@ export default ({ service, pageType, variant }) => {
         retries: 3,
       },
       () => {
-        it('should render an iframe with a valid URL', () => {
+        it('should render a valid media player', () => {
```

Comment:
While we're in here, could we use cy.getPageData instead of cy.request?

Reply:
Looking at the implementation [here](https://github.com/bbc/simorgh/blob/9c1b673a00f789c73d822e89263e1dd692052bbc/cypress/support/commands/application.js#L128), `getPageData` makes a call to the BFF for non-local environments.
Since we don't support On Demand TV in the BFF, would it be appropriate to invoke `getPageData` here?
Also, given the recent work to remove dependence on the BFF for the e2es, would
invoking `getPageData` here be a reinstatement of the said dependency?

Reply:
Oops, I actually meant the `getPageDataFromWindow` which gets it from window.SIMORGH_DATA, which only works on canonical, but that's OK because we don't support AMP on these pages.

Reply:
Ah! Got it.
Thanks for the clarification. Let me give this a go.

Reply:
This is now in place. We've updated [getEpisodeAvailability](https://github.com/bbc/simorgh/pull/11928/files#diff-be0dc0c9e10b2f5281a37e5c0dfc2a0999110ecd8e3fc0c7b4d0125611108e5aR60) to traverse the
schema we receive from `getInitialData`.
`getEpisodeAvailability` will replace [isAvailable](https://github.com/bbc/simorgh/pull/11928/files#diff-be0dc0c9e10b2f5281a37e5c0dfc2a0999110ecd8e3fc0c7b4d0125611108e5aR35) as part of the On Demand Audio migration where it's still in use.